lang/LangPrimSource/SC_ComPort.cpp

```diff
@@ -205,8 +205,25 @@ UDP::UDP(int inPortNum, HandlerType handlerType, int portsToCheck): mPortNum(inP
         }
     }
 
-    boost::asio::socket_base::send_buffer_size option(65536);
-    mUdpSocket.set_option(option);
+    boost::asio::socket_base::send_buffer_size send_buffer_size;
+    mUdpSocket.get_option(send_buffer_size);
```

Comment:
Strictly speaking, `get_option` can also throw. Just wrap the whole code for each buffer in a try-catch statement.

Also, I would post a warning message in the catch-clause.